app/views/layouts/application.html.erb

```diff
@@ -103,7 +103,9 @@
             <% end %>
           </strong>
           <span class="govuk-phase-banner__text">
-            This is a new service – your <%= render MailToSupportComponent.new("feedback") %> will help us to improve it.
+            This is a new service – your
+            <%= link_to "feedback", "https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-VlIdprT6TlEmsOS8-ufjHVUQzgyQVVOSERHWkZYV045RVYxSTBQWlNUSi4u", target: "_blank" %>
```

Comment:
Can we make it a `govuk_link_to`?

And add `rel: "noopener noreferrer"` please, at least if we keep target blank.